com.ca.lsp.cobol/lsp-service-cobol/src/test/java/com/ca/lsp/cobol/service/delegates/references/DocumentOccurrencesTest.java

```diff
@@ -159,6 +175,23 @@ public void testFindDeclarationForParagraph() throws ExecutionException, Interru
     assertContainsRange(locations, range(15, 7, TEST_LENGTH));
   }
 
+  @Test
+  public void testFindMultipleCopybookReferences() throws ExecutionException, InterruptedException {
+    List<? extends Location> locations = invokeReferencesRequest(TEST_COPYBOOK1, true);
+    assertEquals(2, locations.size());
+
+    assertContainsRange(locations, range(20, 12, COPY_LENGTH));
+    assertContainsRange(locations, range(22, 12, COPY_LENGTH));
+  }
+
+  @Test
+  public void testFindSingleCopybookReference() throws ExecutionException, InterruptedException {
+    List<? extends Location> locations = invokeReferencesRequest(TEST_COPYBOOK2, true);
+    assertEquals(1, locations.size());
+
+    assertContainsRange(locations, range(21, 12, COPY_LENGTH));
```

Comment:
could you please provide a negative scenario as well in which you will try to get a reference and there is none